site/daemon.md

```diff
@@ -0,0 +1,73 @@
+---
+title: Flux Daemon
+menu_order: 90
+---
+
+# Summary
+
+Flux daemon (fluxd, aka flux agent) allows automation of application deployments and version control of cluster configuration.
+Version controlling of cluster manifests provides reproducibility and a historical trail of events.
+
+## Flux daemon responsibilities
+
+    A) Continuous Deployment
+	    1.
+        Flux daemon monitors user git repo Kubernetes manifests for changes, which it
+        then deploys to the cluster.
+
+	    2.
+	    Flux daemon monitors container registry for running container image updates.
+        Detection of an image change (running container image tag vs container
+        registry image tag) triggers k8s manifest update, which is committed to the
+        user git repository, then deployed to the Kubernetes cluster.
+
+    B) Deployment approaches
+        1.
+        Automate vs Deautomate
+            Deployment happens automatically when a new image tag is detected.
+            Deautomated deployment will not proceed until manually released (through
+            the UI or the CLI tool fluxctl).
+            
+        2.
+        Lock vs Unlock
+            Deployment is pinned to a particular image tag. New deployment will not
+            proceed upon triggered release.
+    
+# Flags
+
+fluxd requires setup and offers customization though a multitude of flags.
+
+|flag                    | default                       | purpose |
+|------------------------|-------------------------------|---------|
+|--listen -l             | :3030                         | Listen address where /metrics and API will be served|
+|--kubernetes-kubectl    |                               | Optional, explicit path to kubectl tool|
+|--version               | false                         | Get version number|
+|**Git repo & key etc.** |                              ||
+|--git-url               |                               | URL of git repo with Kubernetes manifests; e.g., `git@github.com:weaveworks/flux-example`|
+|--git-branch            | master                        | branch of git repo to use for Kubernetes manifests|
+|--git-path              |                               | path within git repo to locate Kubernetes manifests (relative path)|
+|--git-user              | Weave Flux                    | username to use as git committer|
+|--git-email             | support@weave.works           | email to use as git committer|
+|--git-set-author        | false                         | if set, the author of git commits will reflect the user who initiated the commit and will differ from the git committer|
+|--git-label             |                               | label to keep track of sync progress; overrides both --git-sync-tag and --git-notes-ref|
+|--git-sync-tag          | defaultGitSyncTag             | tag to use to mark sync progress for this cluster (old config, still used if --git-label is not supplied)|
```

Comment:
This should show the actual value, rather than the constant names. Similarly, below.

Reply:
Changed.